mmv1/products/transcoder/JobTemplate.yaml

```diff
@@ -0,0 +1,1137 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: "JobTemplate"
+self_link: "projects/{{project}}/locations/{{location}}/jobTemplates/{{job_template_id}}"
+base_url: "projects/{{project}}/locations/{{location}}/jobTemplates"
+create_url: "projects/{{project}}/locations/{{location}}/jobTemplates?jobTemplateId={{job_template_id}}"
+immutable: true
+description: "Transcoding Job Template Resource"
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    "Transcoder": "https://cloud.google.com/transcoder/docs/"
+  api: "https://cloud.google.com/transcoder/docs/reference/rest/v1/projects.locations.jobTemplates"
+import_format: ["projects/{{project}}/locations/{{location}}/jobTemplates/{{job_template_id}}"]
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "transcoder_job_template_basic"
+    primary_resource_id: "default"
+    vars:
+      job_template_id: "example-job-template"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "transcoder_job_template_overlays"
+    primary_resource_id: "default"
+    vars:
+      job_template_id: "example-job-template"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "transcoder_job_template_encryptions"
+    primary_resource_id: "default"
+    vars:
+      job_template_id: "example-job-template"
+      secret_manager_secret_id: "transcoder-encryption-key"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "transcoder_job_template_pubsub"
+    primary_resource_id: "default"
+    vars:
+      job_template_id: "example-job-template"
+      pub_sub_topic_name: "transcoder-notifications"
+parameters:
+  - !ruby/object:Api::Type::String
+    name: "jobTemplateId"
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      ID to use for the Transcoding job template.
+  - !ruby/object:Api::Type::String
+    name: "location"
+    immutable: true
+    required: true
+    url_param_only: true
+    description: |
+      The location of the transcoding job template resource.
+properties:
+  - !ruby/object:Api::Type::String
+    name: "name"
+    description: |
+      The resource name of the job template.
+    custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
+    output: true
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: "labels"
+    description: |
+      The labels associated with this job template. You can use these to organize and group your job templates.
+  - !ruby/object:Api::Type::NestedObject
+    name: "config"
+    description: |
+      The configuration for this template.
+    default_from_api: true
```

Comment:
Is the `default_from_api: true` set here because of the transcoder API applying a preset template? From the [documentation](https://cloud.google.com/transcoder/docs/how-to/job-templates):

> By default, the Transcoder API applies a preset template called preset/web-hd to populate a job configuration. You can specify your own custom job templates when creating transcoding jobs. A job configuration represents many of the settings you can customize when creating a job.

Reply:
Yes, that’s correct. 

Added, `Job.yaml` also sets it to `true`, but this is because when you reference a template using `template_id`, the API response includes the config and maps it back to the Job resource.